core/src/ibc/client_02/client.rs

```diff
@@ -14,6 +14,7 @@
 // You should have received a copy of the GNU Affero General Public License
```

Comment:
For the commit message: " Adapt ICS02 client to the new core light client algorithm", I don't understand the meaning of the message. Could you explain it to me?

Reply:
@majecty 
Means ICS 02 has to be changed along the change of core light client algorithm.

Reply:
Did you mean that we should change our ICS02 light client implementation along to the change of the light client you changed at the previous commit?
If it is right, please merge it with the light client commit or give a brief description about what was changed in the light client code.

Reply:
I think it's better to just squash